efficientdet/dataloader.py

```diff
@@ -420,7 +421,40 @@ def _prefetch_dataset(filename):
     dataset = dataset.map(
         map_fn, num_parallel_calls=tf.data.experimental.AUTOTUNE)
     dataset = dataset.prefetch(batch_size)
-    dataset = dataset.batch(batch_size, drop_remainder=True)
+    shapes = list(tf.nest.map_structure(lambda item: item.shape,
+                                        dataset.element_spec))
+    shapes[6] = (self._max_instances_per_image,4)
+    shapes[7] = (self._max_instances_per_image,1)
+    shapes[8] = (self._max_instances_per_image,1)
+    shapes[9] = (self._max_instances_per_image,1)
+    shapes = tuple(shapes)
+
+    def make_zero(t):
```

Comment:
These codes look complicated, could you help explain a little bit?

Reply:
I used batch_padded replace pad_to_fixed_size which faster than later.

Reply:
Aha, gotcha. Could you add a simple comment like "Shape and paddings are used for padded_batch."